pio/pio.go

```diff
@@ -135,6 +157,16 @@ func GetConfigPath() (p string, err error) {
 	return
 }
 
+// GetFilesDir is used to get the directory that we store
+// encrypted files in.
+func GetEncryptedFilesDir() (p string, err error) {
+	d, err := GetPassDir()
+	if err == nil {
+		p = filepath.Join(d, EncryptedFileDir)
+	}
+	return
```

Comment:
I would suggest rewriting this without naked returns and using "early return on error" style.

See https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow.

```Go
d, err := GetPassDir()
if err != nil {
    return "", err
}
return filepath.Join(d, EncryptedFileDir), nil
```